libpoolprotocols/getwork/EthGetworkClient.cpp

```diff
@@ -25,22 +24,25 @@ EthGetworkClient::~EthGetworkClient()
 
 void EthGetworkClient::connect()
 {
-	if (m_connection_changed) {
-		stringstream ss;
-		ss <<  "http://" + m_conn->Host() << ':' << m_conn->Port();
-		if (m_conn->Path().length())
-			ss << m_conn->Path();
-		p_client = new ::JsonrpcGetwork(new jsonrpc::HttpClient(ss.str()));
+    stringstream ss;
```

Comment:
Why use a stringstream when a string would do the job?

Use + or += to concatenate strings, and to_string(n) to convert the port #.

Reply:
It's old legacy code. 
Will amend.